src/Microsoft.DotNet.Tools.Repl.Csi/Program.cs

```diff
@@ -14,26 +19,148 @@ public static int Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
-            var app = new CommandLineApplication();
+            var app = new CommandLineApplication(throwOnUnexpectedArg: false);
             app.Name = "dotnet repl csi";
             app.FullName = "C# REPL";
             app.Description = "C# REPL for the .NET platform";
             app.HelpOption("-h|--help");
+
             var script = app.Argument("<SCRIPT>", "The .csx file to run. Defaults to interactive mode.");
+            var framework = app.Option("-f|--framework <FRAMEWORK>", "Compile a specific framework", CommandOptionType.MultipleValue);
```

Comment:
I was originally going to allow multiple frameworks to be specified and to pick the first that matched, but later changed it to match what `dotnet run` was doing. I apparently forgot to change the parameters match and only allow a single value. Fixed now.